src/Components/Recommendation/index.js

```diff
@@ -0,0 +1,23 @@
+import React, { useEffect } from 'react';
+import { useIntl } from 'react-intl';
+
+import routerParams from '@redhat-cloud-services/frontend-components-utilities/RouterParams';
```

Comment:
prefer the use of react-router-dom to routerParams

Reply:
@AllenBW, why the @r-c-s's version is not preferred by you?

Reply:
Wrong r-c-s? I have nothing to do with this project.

Reply:
@gkarat Wrong r-c-s? I have nothing to do with this project.

Reply:
ooops, sorry, wrong mention 😅

Reply:
Another personal preference, in this case ah belief that the package will be kinder to the component lifecycle, it's also mo powerful, (fully featured)